src/Viewer.jsx

```diff
@@ -18,8 +18,8 @@ export const Viewer = ({barColor, formState, language}) => {
 						<Typography sx={{fontWeight: 'bold'}}>{formState.level}</Typography>
 					</Grid>
 				</Grid>
-				<Grid container direction="column" justifyContent="space-between" alignItems="flex-start" height={'calc(100% - 50px)'}	>
-					<Grid item sx={{width:'98%'}} >
+				<Grid container direction="column" justifyContent="space-between" alignItems="flex-start" height={'calc(100% - 50px)'} sx={{minHeight:'300px'}}	>
```

Comment:
Maybe the question that I did was better written here

Reply:
Yes, you can change the minHeight property for the one you want.